dashboard/test/lib/resources_autocomplete_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+class ResourcesAutocompleteTest < ActiveSupport::TestCase
+  # We rely on fulltext indices to be used in this test.
+  # From my experiments, this was the only way to get those created
+  self.use_transactional_tests = false
```

Comment:
Because we rely on the indices, this was the only way I could get this test to pass. I'm definitely open to other strategies here

Reply:
I see you are cleaning up in teardown what you created in setup, so I think we are good here. 👍